pdsfile/pdsfile.py

```
@@ -2629,12 +2631,20 @@ def label_basename(self):
+            ext_guesses = (*uppercase_lbl_ext, *cls.LBL_EXT)
         else:
-            ext_guesses = (cls.LBL_EXT, cls.LBL_EXT.upper())
-
-        rootname = self.basename[:-len(self.extension)]
+            ext_guesses = (*cls.LBL_EXT, *uppercase_lbl_ext)
+
+        # For viewables with label (like f ring browse mosaic), we need to get rid of
+        # _thumb, _full, _med, and _small from the data file names to obtain the correct
+        # root name for the label files.
+        if ('_thumb' in self.basename or '_full' in self.basename or
+            'med' in self.basename or 'small' in self.basename):
```

Flake8 (7.3.0)

[error] visually indented line with same indent as next logical line

(E129)